packages/hw-app-eth/src/services/ledger/index.ts

```diff
@@ -12,7 +12,7 @@ import { getNFTInfo, loadNftPlugin } from "./nfts";
 import { decodeTxInfo } from "../../utils";
 
 const ledgerService: LedgerEthTransactionService = {
-  resolveTransaction: async (rawTxHex, loadConfig) => {
+  resolveTransaction: async (rawTxHex, loadConfig, resolutionConfig) => {
```

Comment:
now, with the resolutionConfig, which is basically user's intent on things to resolve, we can branch the resolution of precisely what is really needed.